### PR TITLE
Airtel

### PR DESCRIPTION
**Pull request by sumit2011**
**Issue:** closes #

**Similarweb rank:**

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by PratyushJayachandran** [comment]
@sumit2011 Please add some details to the Pull Request.
1. Where is the SVG taken from?
2. I see some brand guidelines on their page: https://www.airtel.in/logo-tune. Please include guidelines when available.
3. Why did you chose the stacked version of the logo? Just the logo-mark should be enough unless the brand insists.

**Comment by sumit2011** [comment]
@PratyushJayachandran   hey i am little bit confused that why i am getting error in uploading the svg file.

**Review by adamrusted — CHANGES_REQUESTED** [review]
I'm seeing a slight difference at the sides of the icon, potentially from the shape being stretched to fill the `24*24` viewport?
Could you try building from source again with a precision of 4 decimals (`npx svgo icons/airtel.svg -p 4` and see if that gives us a better result @sumit2011?

image

File: _data/simple-icons.json [hunk 1]
```diff
@@ -435,6 +435,11 @@
             "hex": "18BFFF",
             "source": "https://airtable.com/press"
         },
+        {
+            "title": "Airtel",
+            "hex": "FF0000",
+            "source": "https://www.airtel.in"
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:441]:
```suggestion
            "hex": "E40000",
            "source": "https://www.airtel.in/logo-tune"
```

File: icons/airtel.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...71c.968.029 2.008.651 2.051 1.645.035.747-.503 1.269-.892 1.651l-.071.068c-.317.305-.629.585-.933.868-.528.481-1.032.939-1.527 1.511 0 0-1.358 1.918.878 1.332.125-.034.259-.091.415-.148l.079-.02c.531-.199 1.034-.475 1.478-.713l.091-.043c.957-.507 2.032-1.257 3.199-2.22 2.284-1.913 3.939-3.894 4.894-5.918.699-1.482.828-3.097.311-4.126-.291-.586-.793-.96-1.47-1.114a2.988 2.988 0 0 0-.666-.072c-.459 0-.969.093-1.549.289-2.219.755-4.202 2.046-6.113 3.294l-.233.147c-.23.165-.467.314-.699.461-.083.047-.17.11-.254.169-1.351.797-3.317 1.748-5.106 1.748-.877 0-1.628-.234-2.222-.685-.673-.505-1.024-1.197-1.067-2.038-.106-1.974 1.582-4.31 3.054-5.697.162-.142.325-.278.495-.408.166-.132.334-.266.504-.398.171-.125.336-.25.506-.373.176-.122.354-.241.534-.359l.069-.042c.022-.016.047-.028.063-.044l.073-.042c.023-.015.045-.027.062-.041l.077-.057c.031-.014.055-.029.078-.042l.077-.049a.646.646 0 0 1 .087-.05l.008-.015.113-.069C9.306 1.138 12.61 0 15.62 0c3.297 0 5.043 1.369 5.206 1.495 1.403.967 2.364 2.242 2.859 3.788.67 2.129.122 4.393-.269 5.558-.971 2.862-2.653 5.212-5.46 7.602-1.267 1.079-2.498 1.991-3.763 2.764l-.318.192c-1.197.738-2.432 1.502-3.991 2.045-1.047.365-1.973.556-2.751.556Z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/airtel.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Airtel</title><path d="M7.137 23.862c.79 0 1.708-.19 2.751-.554 1.55-.538 2.784-1.281 3.986-2.009l.316-.205a29.733 29.733 0 0 0 3.764-2.72 16.574 16.574 0 0 0 5.457-7.529c.395-1.138.949-3.384.268-5.487a7.117 7.117 0 0 0-2.862-3.749c-.158-.126-1.898-1.47-5.203-1.47-3.005 0-6.31 1.107-9.806 3.32l-.11.08-.317.205a20.133 20.133 0 0 0-2.309 1.693C1.585 6.813-.091 9.106.004 11.067c.031.79.427 1.534 1.075 2.008a3.472 3.472 0 0 0 2.214.68c1.803 0 3.765-.948 5.109-1.74l.253-.157.696-.443.237-.158c1.898-1.234 3.875-2.515 6.105-3.258a5.255 5.255 0 0 1 1.55-.285 3.163 3.163 0 0 1 .664.08 2.112 2.112 0 0 1 1.47 1.106c.523 1.012.396 2.61-.316 4.08a17.871 17.871 0 0 1-4.887 5.836 19.488 19.488 0 0 1-3.194 2.215l-.095.031a9.634 9.634 0 0 1-1.471.696l-.08.032-.41.158c-2.23.57-.87-1.329-.87-1.329.474-.537.98-1.028 1.518-1.502.316-.269.633-.554.933-.854l.064-.063c.395-.38.933-.902.901-1.645-.047-.98-1.075-1.582-2.056-1.613h-.063c-.95 0-1.819.522-2.404.98a7.27 7.27 0 0 0-1.598 1.74c-.6.901-1.85 3.226-.632 5.077.49.743 1.313 1.123 2.42 1.123z"/></svg>
```

**Review by PeterShaggyNoble — DISMISSED** [review]

**Review by adamrusted — APPROVED** [review]

**Pull request merged**